src/helpers/ens.ts

```diff
@@ -507,7 +518,6 @@ const accentColorAtom = atom({
 });
 
 export {
```

Comment:
Feel like we should export the functions directly (it's better for autocompletion). Thoughts?

Reply:
yeah I agree